woodwork/utils.py

```diff
@@ -54,6 +54,36 @@ def import_or_none(library):
         return None
 
 
+def ww_concat(base, to_add):
```

Comment:
Should this be a private testing utility instead? Seems like we are not using this anywhere outside of the tests and I'm thinking this is not a function we would ever expect a Woodwork user to use.

Also, the name `ww_concat` makes me thing this is a function for concatenating Woodwork objects...maybe we should consider a different name?

Reply:
Regarding the name, how does `_concat_dataframe_or_series` sound? It is a little verbose but more descriptive.

Reply:
I'm fine with that name

Reply:
Interestingly, it was not my intent to alter test methods only. It just so happened that all uses of `append` (at least the ones that are apparent to me to be deprecated) were in testing functions. Would making this a test utility entail moving this function to another part of the codebase?

Would this be something useful in other parts of Woodwork?

For example, I noticed a prospective use here: 
https://github.com/alteryx/woodwork/blob/e5b1c66bbadf1cfd07de4ae6bef25f4226addac4/woodwork/utils.py#L528

@thehomebrewnerd @ParthivNaresh @bchen1116

Reply:
That sounds reasonable to me. I think having it integrated into other parts of woodwork where we might use concat/append would be helpful, although that seems out of scope for this. Maybe it'd be useful to leave this as a testing util and file an issue to look into potential uses for it in woodwork general at a later point?

Reply:
Yeah, I agree with @bchen1116. In order to use your new function in the `concat_columns` function, you would need to make some more updates to the function to allow for changing the concat axis. If the only place it would be used is in that one spot, I'm not sure that is worthwhile.

Reply:
Sounds good! To make sure I am understanding this correctly, we would want to to move this to `woodwork/tests/testing_utils/concat.py` and then create a follow-up issue to track potentially adding more functionality to this method (or adding a new method).

Reply:
I think we can just add it to the existing `table_utils.py` file rather than creating a new one just for this utility function. Yes, you can then create a new issue to look into further updates.